src/components/VideoSelector/VideoSelector.tsx

```diff
@@ -1,11 +1,16 @@
+'use client';
 import React, { useRef, useState } from 'react';
 import './VideoSelector.css';
-import pressplay from '../../playicon.png';
+import pressplay from '@/assets/playicon.png';
+import { useVideoContext } from '../VideoContextProvider/VideoContextProvider';
```

Comment:
for future work, I would say contextProviders should not be written in the components folder. Because they aren't components, it makes more sense to keep them in a `helpers` folder or even a `providers` folder